packages/react-charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -159,6 +160,12 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
         startFromX: 0,
       });
     }
+    for (let i = 0; i < this.props.points.length; i += 1) {
```

Comment:
Use array.some to iterate over the points as used in other places.
Also use the points[i].y % 1 === 0 condition to check for integer values.

Reply:
Ok